Composer/packages/client/src/constants.ts

```diff
@@ -420,7 +420,8 @@ export const nameRegex = /^[a-zA-Z0-9-_]+$/;
 
 export const nameRegexV2 = /^[a-zA-Z0-9_]+$/;
 
-export const invalidNameCharRegex = /[^a-z^A-Z^0-9^_]/g;
+export const invalidNameCharRegex = /[^a-zA-Z0-9-_]/g;
+export const invalidNameCharRegexV2 = /[^a-zA-Z0-9-_]/g;
```

Comment:
These look identical, and I don't see where you use the V2 one.

Reply:
That's a good catch. I just copied but forgot to update. I need to remove the - from the non V2 one. As for not being used-- the original variable isn't used anywhere, but I thought I'd add the V2 one to differentiate it in the future.